.travis.yml

```diff
@@ -1,4 +1,5 @@
 ---
+dist: bionic
```

Comment:
Do we need this change or is it just because it is possible?

Reply:
I add that for the future support of podman. Yes. I should move that to another PR.

Reply:
Thank you. Added the commit to #577 since it matches the topic and adjusted the commit message (hope it is ok for you).